tests/scripts/test-ci.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+function push() {
+    arm64_Master_Image=$(docker images --format "{{.Repository}}:{{.Tag}}" | grep "arm64:latest" | head -n 1)
+    docker tag "$arm64_Master_Image" quay.io/rook/rook-arm64:master
+    docker push quay.io/rook/rook-arm64:master
```

Comment:
If we're in a release branch, we don't want to push the master image. We should only be pushing either to the master tag, release branch tag, or to the release version tag. Maybe the tag name could be passed as an arg from the makefile, so you don't have to compute it again.

Reply:
thanks for suggestion since script is removed, this is not valid anymore.